fashion-messaging-app-api/models/searchResult.js

```diff
@@ -4,7 +4,7 @@ import { sequelize } from '../database.js';
 export const SearchResult = sequelize.define('SearchResult', {
     userId: {
         type: DataTypes.INTEGER,
-        allowNull: false,
+        allowNull: true,
```

Comment:
I changed this to true while I was debugging the issue of the userid returning null in my database. This allowed me to observe the database's behavior in handling the situation when a user is not logged in and it saves my code from breaking each time the userid returns null